src/NServiceBus.Core/Transports/TransportComponent.cs

```diff
@@ -145,7 +145,7 @@ public string ToTransportAddress(LogicalAddress logicalAddress)
 
             public bool ReceivingEnabled { get; }
 
-            public readonly TransportInfrastructure transportInfrastructure;
+            public readonly TransportInfrastructure TransportInfrastructure;
```

Comment:
fields should be lower-case?

Reply:
Fixed

@SeanFeldman we did this on purpose since the field have to be exposed for the component to use later but should not be used by any other components

Reply:
In that case I shouldn't be using it in the ReceiveComponent.

Reply:
Good point, fixed